lib/rails_multitenant/global_context_registry.rb

```diff
@@ -87,7 +87,7 @@ def replace_registry(registry)
 
     # Run a block of code that disregards scoping during read queries
     def with_unscoped_queries
-      with_merged_registry(__use_unscoped_queries: true) do
+      with_merged_registry(__use_unscoped_queries: true) do # rubocop:disable Style/ExplicitBlockArgument
```

Comment:
Why disable this cop here?

Reply:
There's definitely performance implications to the recommended style. I'm considering opening a PR against salsify_rubocop for making this optional or disabling. https://github.com/JuanitoFatas/fast-ruby/blob/master/code/proc-and-block/proc-call-vs-yield.rb

```
Warming up --------------------------------------
          block.call   700.776k i/100ms
       block + yield   709.674k i/100ms
        unused block   858.726k i/100ms
               yield     1.015M i/100ms
Calculating -------------------------------------
          block.call      6.819M (± 6.2%) i/s -     34.338M in   5.056390s
       block + yield      7.103M (±10.6%) i/s -     35.484M in   5.059581s
        unused block      8.768M (±11.7%) i/s -     43.795M in   5.076244s
               yield      9.561M (± 8.6%) i/s -     47.686M in   5.032351s

Comparison:
               yield:  9560970.9 i/s
        unused block:  8767778.8 i/s - same-ish: difference falls within error
       block + yield:  7103240.3 i/s - 1.35x  (± 0.00) slower
          block.call:  6818701.2 i/s - 1.40x  (± 0.00) slower
```

Reply:
I thought the performance characteristics of this changed in more recent versions of the MRI but `block.call` still looks significantly slower in Ruby 3.1. Mind adding a comment do the code as to why we're disabling this cop here?